app/src/main/java/com/codepath/travel/activities/CreateStoryActivity.java

```diff
@@ -147,6 +171,38 @@ private void setUpRecyclerView() {
         mItemTouchHelper.attachToRecyclerView(rvStoryPlaces);
     }
 
+    private void addSelectedPlaceInTrip(Place place, String photoReference) {
+        StoryPlace storyPlace = new StoryPlace(mNewTrip, place);
+        storyPlace.setThumbnail(photoReference);
```

Comment:
So the cover pic already has a default link which is used for showing cover pic of trips which is why I introduced a new field thumbnail for the suggested places. Also it kinda makes more sense as it is not essentially a "cover pic" more than a "thumbnail"